Add did key spec 7 validators (#63)
digitalbazaar/did-io: CHANGELOG.md

# did-io ChangeLog

## 3.0.0 -

### Changed

> - **BREAKING**: `parseDid` not longer returns prefix. Returns method instead.

Reviewer: ```suggestion
- **BREAKING**: `parseDid` no longer returns `prefix`. Returns `method` instead.
```

> - **BREAKING**: `parseDid` no longer returns `prefix`. Returns `method` instead.

Reviewer: ```suggestion
- **BREAKING**: `parseDid` changed to return `scheme` and `method` instead of
  only `prefix`.
```

### Added

> - **BREAKING**: `CachedResolver` now validates dids before resolving them.

Reviewer: ```suggestion
- **BREAKING**: `CachedResolver` now validates DIDs before resolving them.
```
